libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -828,6 +828,25 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         )
     }
 
+    fun previewNavigationRoutes(
+        routes: List<NavigationRoute>,
+        initialLegIndex: Int = 0,
+    ) {
+        //TODO: parse alternatives metadata using NN and set it route alternatives controller
+        directionsSession.setRoutes(routes, initialLegIndex, RoutesExtra.ROUTES_UPDATE_REASON_NEW)
+    }
+
+    fun setPreviewedRoute() {
+        val routes = directionsSession.routes
+        if (routes.isNotEmpty()) {
+            internalSetNavigationRoutes(
```

Comment:
How is it gonna affect billing? It receives a route in `MapboxNavigation#setNavigationRoutes`, not in `internalSetNavigationRoutes `.

Reply:
thanks, but billing is not supposed to work in this prototype